FinSearchUnified/Tests/Helper/StaticHelperTest.php

```diff
@@ -600,60 +599,12 @@ public function testCleanStringMethod($text, $expected, $errorMessage)
      */
     public function testBuildCategoryName($categoryId, $category, $expected)
     {
-        $categoryModel = $this->categoryResource->update($categoryId, [
-            'name' => $category
-        ]);
-        $this->assertInstanceOf(Category::class, $categoryModel);
-
-        $this->updateParentCategoryName($categoryModel->getParent(), false);
-
+        $categoryModel = $this->categoryResource->update($categoryId, ['name' => $category]);
```

Comment:
Why cant we reset our changes to the database after a test has been completed e.g. `tearDown`?

Reply:
It is specific to only this case. We don't want to update the database for every case in this class.